Doc/library/ftplib.rst

```diff
@@ -449,17 +451,51 @@ FTP_TLS objects
 
    A :class:`FTP` subclass which adds TLS support to FTP as described in
    :rfc:`4217`.
-   Connect as usual to port 21 implicitly securing the FTP control connection
-   before authenticating. Securing the data connection requires the user to
-   explicitly ask for it by calling the :meth:`prot_p` method.  *context*
-   is a :class:`ssl.SSLContext` object which allows bundling SSL configuration
-   options, certificates and private keys into a single (potentially
-   long-lived) structure.  Please read :ref:`ssl-security` for best practices.
```

Comment:
Do we not need this "Please read..." sentence?

Reply:
It's still there; in the param docs.